Source/UI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -18,9 +18,12 @@ public enum PaymentFormFactory {
                                                 billingFormStyle: style.billingFormStyle,
                                                 supportedSchemes: configuration.supportedSchemes)
         viewModel.preventDuplicateCardholderInput()
-
+        
+        
         let viewController = FramesPaymentViewController(viewModel: viewModel)
-        viewModel.cardTokenRequested = completionHandler
+        viewModel.cardTokenRequested = { result in
+            completionHandler(result)
+        }
         logger.log(.paymentFormInitialised(environment: configuration.environment))
         if #available(iOS 13.0, *) {
             viewController.isModalInPresentation = true
```

Comment:
we can have no diff here

Reply:
can we literally have this file not in this PR. Not the spaces or newline diff as well? Easier to track and understand that way